full.async/src/full/async.clj

```diff
@@ -62,6 +68,26 @@
              (recur (dec retries#)))
            res#)))))
 
+(defmacro go-try> [err-chan & body]
```

Comment:
You don't know whether the first argument expands to a channel at macro-expansion time, do you?